src/qt/transactionrecord.cpp

```diff
@@ -162,7 +162,7 @@ QList<TransactionRecord> TransactionRecord::decomposeTransaction(const interface
     return parts;
 }
 
-void TransactionRecord::updateStatus(const interfaces::WalletTxStatus& wtx, int numBlocks, int64_t block_time)
+void TransactionRecord::updateStatus(const interfaces::WalletTxStatus& wtx, uint256 block_hash, int numBlocks, int64_t block_time)
```

Comment:
In commit "[WalletModel] Balance and Txrecord status change polling check based on the block hash and not the height." (997129bdb45c01390bffce991b680d58c9e7d18b)

Maybe use `const uint256&` type to avoid copy here, similarly for other functions in this commit

Reply:
done